packages/loot-core/src/shared/rules.js

```diff
@@ -57,8 +57,6 @@ export function mapField(field, opts) {
       return 'amount (inflow)';
     case 'amount-outflow':
       return 'amount (outflow)';
-    case 'cleared':
-      return 'cleared';
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Why was this removed?

Reply:
oh, this just makes the name "pretty". and since var name matches what we return - there's no point in this. :)